gambit/README.md

```diff
@@ -0,0 +1,35 @@
+# Gambit MIMIC Integration
+
+## Overview
+
+Gambit MIMIC is a dynamic, real-time, high-performance SNMP, Netflow, sFlow Simulator. All Netflow and sFlow exports are recorded by the simulator and replayed with flexible customization toward Datadog agent(s) and presented with all existing available metrics of Datadog, whether built-in or Datadog marketplace integrated.
```

Comment:
```suggestion
[Gambit MIMIC][1] is a dynamic, real-time, high-performance SNMP, Netflow, and sFlow simulator. The simulator records all Netflow and sFlow exports, which are replayed with flexible customization toward Datadog agents. The exports are presented alongside all available Datadog metrics, including built-in and Datadog Marketplace integrated.
```

Reply:
I'm not sure if I've correctly interpreted the meanings of everything here, especially at the start of the last sentence.